src/curated-corpus/components/ApprovedItemForm/ApprovedItemForm.tsx

```diff
@@ -550,13 +555,11 @@ export const ApprovedItemForm: React.FC<
       </Grid>
       <Box display="none">
         <TextField
-          type="hidden"
```

Comment:
removed this as it was preventing the tests from finding the fields. `display="none"` on the wrapping element keeps them out of view.